src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -164,7 +172,7 @@ public async Task<IActionResult> List(ListContentsViewModel model, PagerParamete
                 {
                     if (ctd.GetSettings<ContentTypeSettings>().Listable)
                     {
-                        var authorized = await _authorizationService.AuthorizeAsync(User, Permissions.EditContent, await _contentManager.NewAsync(ctd.Name));
+                    var authorized = await _authorizationService.AuthorizeAsync(User, CommonPermissions.EditContent, await _contentManager.NewAsync(ctd.Name));
```

Comment:
```suggestion
  // We want to list the content item if the user can edit their own items at least. It might display content items the user won't be able to edit though.
                    var authorized = await _authorizationService.AuthorizeAsync(User, CommonPermissions.EditContent, await _contentManager.NewAsync(ctd.Name) { Owner =  User.Username });
```

Reply:
Doesn't make any difference with the changes I made in the ContentTypeAuthorizationHandler. Unless you want me to revert the changes in the handler. Here passing the owner or not doesn't change the result.

Reply:
Doesn't make any difference with the changes I made in the ContentTypeAuthorizationHandler. Unless you want me to revert the changes in the handler. Here passing the owner or not doesn't change the result.

> and are you using await _contentManager.NewAsync() or are you using new ContentItem()?

Only difference is that we are calling the events of the DefaultContentManager this way. Which I don't like, but gives the same result for us.